docs/user-manual/en/amqp-broker-connections.md

```diff
@@ -0,0 +1,238 @@
+# Broker Connections
+
+Instead of waiting for clients to connect, a broker can initiate a connection to another endpoint on a specific protocol.
+
+Currently, this feature supports only the AMQP protocol. However, in the future, it might be expanded to other protocols.
+
+You configure broker connections using a `<broker-connections>` element in the `broker.xml` configuration file.
+
+*Important*: This is a new feature. That means test your usecases well and report eventual issues through users@activemq.apache.org, as the community is actively working to resolve any issues around Broker Connections.
```

Comment:
I don't think this 'important' is necessary, it is true of all features all of the time.

Reply:
I thought I had removed that already.. thanks!